notebooks/120-tf2-object-detection-to-openvino/120-tf2-object-detection-to-openvino.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<p>can we provide something like next steps with links on other tutorials like async inference, preprocessing API and so on?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Next steps section added